src/api/include/migraphx/migraphx.hpp

```diff
@@ -979,6 +979,23 @@ struct program : MIGRAPHX_HANDLE_BASE(program)
         return arguments(pout, own{});
     }
 
+// Needed to enable stream sync functionality at compile time with onnxruntime
+// Helps deal with compatability so we dont break execution provider.
+#define MIGRAPHX_STREAM_SYNC
+    template <class Stream>
+    /// Overloaded to allow for execution_environment input
```

Comment:
This comment should be updated, there is no `execution_environment` in the API.

Reply:
Yup done.